packages/suite/src/hooks/wallet/useRbfForm.ts

```diff
@@ -34,16 +35,26 @@ const getBitcoinFeeInfo = (info: FeeInfo, feeRate: string) => {
 
 const getEthereumFeeInfo = (info: FeeInfo, gasPrice: string) => {
     const current = new BigNumber(gasPrice);
-    const minFee = current.plus(1);
+    const minFeeFromNetwork = new BigNumber(
+        fromWei(info.levels[0].feePerUnit, 'gwei'),
+    ).integerValue(BigNumber.ROUND_FLOOR);
+
+    const getMinFee = () => {
+        if (minFeeFromNetwork.toNumber() <= current.toNumber()) {
+            return current.plus(1);
+        }
+        return minFeeFromNetwork;
+    };
+
     // increase FeeLevel only if it's lower than predefined
     const levels = getFeeLevels('ethereum', info).map(l => ({
         ...l,
-        feePerUnit: current.lte(gasPrice) ? minFee.toString() : gasPrice,
+        feePerUnit: getMinFee().toString(),
```

Comment:
why call `getMinFee()` twice? 
more effective would be if you calculate it and assign once
```
const minFee = getMinFee();

// and used later
feePerUnit: minFee.toString()
...
minFee: minFee.toNumber()

```